blueprints/component/index.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
```

Comment:
do we really need this files? if we won't create them won't they be taken from `ember-source`?

Reply:
Cool! Great, yes, you're right. I can remove those. I wasn't sure if/how things get picked up at root level.

Reply:
@chilicoder hmm I still need the template files it appears, and the component file needs `importTemplate` which is referenced in the template. Otherwise, it's not found. Probably need to find a more sophisticated way to extending these from ember-source...

Reply:
strange.. will check it later

Reply:
Yeah. I have the same issue. When I have time will debug `ember generate` command

Reply:
@allthesignals I found how to solve this. inside of your `component/index.js` write
```
const Blueprint = require('ember-source/blueprints/component');

module.exports = Blueprint;
```

Reply:
@chilicoder sweet! I'll update. Thank you :) figured it was just the classic AMD stuff